platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -802,6 +802,17 @@ + (instancetype)expressionWithMGLJSONObject:(id)object {
         } else if ([op isEqualToString:@"to-string"] || [op isEqualToString:@"string"]) {
             NSExpression *operand = [NSExpression expressionWithMGLJSONObject:argumentObjects.firstObject];
             return [NSExpression expressionWithFormat:@"CAST(%@, 'NSString')", operand];
+        } else if ([op isEqualToString:@"to-color"]) {
+            NSExpression *operand = [NSExpression expressionWithMGLJSONObject:argumentObjects.firstObject];
```

Comment:
`to-color` can take multiple arguments, similar to `to-number`. I’m not a huge fan of `mgl_numberWithFallbackValues:`, so I won’t request an `mgl_colorWithFallbackValues:`. However, if there is more than one argument to this expression, we should at least fall back to `MGL_FUNCTION()` instead of losing all the fallback values in translation.